starter-code/iteration-1/main.js

```diff
@@ -0,0 +1,34 @@
+// Iteration 1
+// 1. Create an array of strings
+var myArray = [];
+// 2. Create a function to add a task in to the array, call it `addTask`, 
```

Comment:
Even though the function has to accept a string, they want the function itself to return a Number. Since the bottom has a console log that tells us "number" -> `console.log("Number of items:", addTask('Eat pizza 🍕 yummy!!!'));`

```js
let addTask = function (task: string): number {
  this.myArray.push(task);

  console.log("=========== NEW INSERTION ===========");
  console.log(`Task "${task}" inserted in the list`);

  return this.myArray.length;
}
``` 

... So you had to add `: number`  after your parameter and then had to return a number since thats the "type" we just told it expect, which is why we return the ".length"